src/Components/GenreSelector/GenreSelector.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+const title = 'Genre Selector';
+
+export class GenreSelector extends React.Component {
```

Comment:
You can write components using class syntax, of course
but nowadays, it is being used very rarely
I'd recommend using hooks and class components instead

(in the task mentioned writing Counter component with class syntax):
![image](https://github.com/AlieksieievOU/react-netflix-roulette/assets/40394400/70b784a2-dd3f-472a-b2a3-939ad9927563)